deployed/matic/abi/contracts/Core/interface/IJobManager.sol/IJobManager.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
do we need all these files? these should be in .gitignore as they are generated during deployment right?
This is cluttering the commit.

Reply:
These are meant to be pushed for now. Replacing with npm deployment in a future commit.

Reply:
We commit these ABIs as part of deployment as of now - https://github.com/razor-network/contracts/blob/master/deployed/goerli/abi/contracts/Delegator.sol/Delegator.json

Reply:
Resolving this now @hrishikeshio since it is expected for now.